src/r5py/r5/base_travel_time_matrix_computer.py

```diff
@@ -71,7 +72,9 @@ def __init__(
             ``max_time_cycling``, ``max_time_driving``, ``speed_cycling``, ``speed_walking``,
             ``max_public_transport_rides``, ``max_bicycle_traffic_stress``
         """
-        if not isinstance(transport_network, TransportNetwork):
+        if not isinstance(transport_network, TransportNetwork) and not isinstance(
+            transport_network, CustomCostTransportNetwork
+        ):
```

Comment:
this could be written as 
```
if not isinstance(transport_network, (TransportNetwork, CustomTransportNetwork)
```
BUT, **more importantly**, does not have to be changed at all, as your `CustomTransportNetwork` inherits from `TransportNetwork` and thus is an instance of its type ;)

Reply:
oh didn't know you could tuplify isinstance's like that, how cool! And how silly of me, good point that it will always be TransportNetwork! Removing that obsolete isinstance check!